plugins/modules/mysql_replication.py

```diff
@@ -490,8 +506,14 @@ def main():
     # "REPLICA" must be used instead of "SLAVE"
     if uses_replica_terminology(cursor):
         replica_term = 'REPLICA'
+        if master_use_gtid == 'slave_pos':
+            module.warn('master_use_gtid "slave_pos" value is deprecated. '
```

Comment:
Why not use `module.deprecate()` and disallow `slave_pos` in the major release?

Reply:
@felixfontein 1) I wouldn't like to deprecate it in observable future 2)  it's hard to predict the exact version 3) I'm waiting for anybody persuades me that it's worth to deprecate and disallow eventually :)
What do you folks think?

Reply:
@felixfontein do you mean exactly `slave_pos` or all the expressions?

Reply:
@Andersson007 I mean for all of them :)

What do you mean by "it's hard to predict the exact version"?

Reply:
@felixfontein I wan't clear sorry, I mean if we say it'll be community.mysql 2.0 or 3.0 or whatever, I'm not sure it will happen. This collection is too small and breaking changes aren't introduced regularly, so it's hard to choose version and to correlate it with a term.
Let's think.. I think we could:
1) change releasing policy to release from special branches (now we release from `main`)
2) introduce only change of the messages I mentioned in the fragment in the next major release (2.0.0), say in 3-6 months
3) and then remove the options I added the aliases to in 3.0.0 in 6-12 months since now

What do you folks think?

Reply:
I think the process makes sense, although I don't quite understand the impact of releasing from special branches.
Also, in a shameless wedge to push my own agenda, I think the process applies as well to the issue outlined in #101

Reply:
@Jorge-Rodriguez briefly it's more complicated than just to release from main. We'll have to backport things to release branches, e.g. to `stable-1`, release fix versions for all supported branches, etc. Anyway, we have no option with `deprecated` terminology - it must be replaced, so will release from branches, no problem.